src/data_access/ConversationDataAccessObject.java

```diff
@@ -12,37 +13,23 @@
 
 public class ConversationDataAccessObject implements ConversationDataAccessInterface {
 
-    // This is the csv file
-    private final File csvFile;
-    // This map is for the headers of our csv file // TODO: decide how to organize this
-    private final Map<String, Integer> headers = new LinkedHashMap<>();
-
-    // PromptGenerator entity which we'll build from csv flie
+    // We are calling it the ConversationDataAccessObject because we originally only wanted the DAO to save
+    // ConversationHistory but then people forgot and treated the DAO like a PromptGenerator DAO so now we are keeping
+    // the name without refactoring other code because we have bigger issues to deal with for now.
+    // We recognize that this is contributing to the technical debt of the project.
     private PromptGenerator promptGenerator;
```

Comment:
This is absolutely fantastic very clean code, the csv stuff will be in our git history for the memroes